services/bitbucket/bitbucket-pull-request.service.js

```diff
@@ -14,14 +15,65 @@ function pullRequestClassGenerator(raw) {
   const badgeSuffix = raw ? '' : ' open'
 
   return class BitbucketPullRequests extends BaseJsonService {
-    async fetch({ user, repo }) {
-      const url = `https://bitbucket.org/api/2.0/repositories/${user}/${repo}/pullrequests/`
-      return this._requestJson({
-        url,
+    async fetchCloud({ args, user, repo }) {
+      args.url = `https://bitbucket.org/api/2.0/repositories/${user}/${repo}/pullrequests/`
+      args.options = { qs: { state: 'OPEN', limit: 0 } }
+
+      if (
+        serverSecrets &&
```

Comment:
Let's remove the truthy check on `serverSecrets` here (see #2606 and #2634 for more info)